### PR TITLE
Update web messenger screenshots

### PR DESCRIPTION
**Pull request by Mario54**
@jugarrit @dannytranlx @mspensieri @chloepouprom

### CODE REVIEW

**Comment by mspensieri** [comment]
👍

**Comment by dannytranlx** [comment]
👍

File: source/includes/customization/_string_customization_js.md [hunk 1]
```diff
@@ -1,30 +1,14 @@
 ## Strings customization
 
-Smooch lets you customize any strings it displays by overwriting its keys. Simply add the `customText` key in your `Smooch.init()` call and specify new values for the keys used in Smooch. You can find all available keys by browsing to the `defaultText` object in the source. If something is between `{}`, it's a variable and needs to stay in your customized text if you want to use it.
+Smooch lets you customize any strings it displays by overwriting its keys. Simply add the `customText` key in your `Smooch.init()` call and specify new values for the keys used in Smooch. You can find all available keys [here](https://github.com/smooch/smooch-js/blob/integration/src/js/reducers/ui-reducer.js#L6). If some text is between `{}`, or if there is an html tag such as `<a>`, it needs to stay in your customized text.
```
Comment (mspensieri) [inline source/includes/customization/_string_customization_js.md]: can you point to `/blob/master` instead?

Comment (mspensieri) [inline source/includes/customization/_string_customization_js.md]: `https://github.com/smooch/smooch-js/blob/master/src/js/reducers/ui-reducer.js#L6`

Comment (Mario54) [inline source/includes/customization/_string_customization_js.md]: oh yeah!

**Comment by Go-Fred** [comment]
Should we have a couple more examples instead of one?

**Comment by dannytranlx** [comment]
👍

**Comment by Go-Fred** [comment]
👍

**Pull request merged**